toolchains/iOS.framework.cmake

```diff
@@ -47,7 +47,10 @@ set(FRAMEWORK_HEADERS
     ${PROJECT_SOURCE_DIR}/ios/framework/TangramMap.h
     ${PROJECT_SOURCE_DIR}/ios/src/TGMapViewController.h)
 
-add_library(${FRAMEWORK_NAME} SHARED ${SOURCES} ${HEADERS})
+add_bundle_resources(RESOURCES "${PROJECT_SOURCE_DIR}/scenes" ".")
```

Comment:
@msmollin Hm I see, Can you try the following:
- comment out this line
- regenerate the whole project
- build the framework again

This line injects the scene resources into the framework under `TangramMap/framework`, to see if your issue is still happening?